nixnet/system/system.py

```diff
@@ -68,6 +70,11 @@ def close(self):
         self._handle = None
 
     @property
+    def databases(self):
```

Comment:
I was getting a flake8 error when trying to have return type: databases.Databases

So I removed it. It may have to do with 'databases' being overloaded